shared/teams/index.js

```diff
@@ -3,7 +3,25 @@ import Render from './render'
 import pausableConnect from '../util/pausable-connect'
 import openURL from '../util/open-url'
 
-const mapStateToProps = state => ({})
+import type {TypedState} from '../constants/reducer'
+
+const mapStateToProps = (state: TypedState) => {
+  // TODO: Figure out better way to get list of teams -- this won't
+  // work until the inbox is loaded.
```

Comment:
It sounds like I'll have to plumb up the team member count from the service, so can probably put all the info we need there too

Reply:
Done, hooked up to teamList RPC